app/views/social_previews/tag.html.erb

```diff
@@ -101,8 +79,15 @@
         <%= @tag.pretty_name %>
       </h2>
     <% end %>
-    <div class="preview-dev-logo">
-      <%= inline_svg_tag("devplain.svg", class: "logo", size: "9vw * 9vw", aria: false, title: "DEV logo") %>
+    <div class="badge-images">
+      <% if @tag.badge_id %>
+        <%# Use a seed in Random so we don't break cache on every render %>
+        <% @badge = Badge.find(@tag.badge_id) %>
+        <% not_so_rand = Random.new(@badge.id) %>
+        <img src="<%= @badge.badge_image_url %>" style="transform: rotate(<%= not_so_rand.rand(-6..6) %>deg);" />
```

Comment:
could we move this logic up in the controller and just pass the badge and the rotation value to the view?

Reply:
Yeah. I'm going to push this through because we wanted the fix due toe #twiliohackathon getting cut off, and that's going live today. But we can move this around soon.